[2022-06-27 13:32:58,305] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-05-24T00:00:00+00:00 [queued]>
[2022-06-27 13:32:58,358] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_A scheduled__2022-05-24T00:00:00+00:00 [queued]>
[2022-06-27 13:32:58,360] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:58,365] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:32:58,375] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:58,477] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_A> on 2022-05-24 00:00:00+00:00
[2022-06-27 13:32:58,520] {standard_task_runner.py:52} INFO - Started process 58244 to run task
[2022-06-27 13:32:58,559] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_A', 'scheduled__2022-05-24T00:00:00+00:00', '--job-id', '714', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmpkszvjqzc', '--error-file', '/tmp/tmpecgi1zbj']
[2022-06-27 13:32:58,593] {standard_task_runner.py:80} INFO - Job 714: Subtask model_A
[2022-06-27 13:32:58,755] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_A scheduled__2022-05-24T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:32:58,987] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_A
AIRFLOW_CTX_EXECUTION_DATE=2022-05-24T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-24T00:00:00+00:00
[2022-06-27 13:32:58,989] {python.py:173} INFO - Done. Returned value was: 10
[2022-06-27 13:32:59,043] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_A, execution_date=20220524T000000, start_date=20220627T133258, end_date=20220627T133259
[2022-06-27 13:32:59,142] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:32:59,240] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
